workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -97,6 +105,16 @@ export const WorkflowInstancePageContent: React.FC<{
       delete instanceVariables.result;
     }
   }
+  const workflowId = assessedInstance.instance.processId;
+  const instanceId = assessedInstance.instance.id;
+  const { value } = useAsync(async (): Promise<InputSchemaResponseDTO> => {
+    const res = await orchestratorApi.getWorkflowDataInputSchema(
+      workflowId,
+      instanceId,
+    );
+    return res.data;
+  }, [orchestratorApi, workflowId]);
+  const inputs = value?.data;
```

Comment:
nice way to do it
Indeed if you pass the instance id to the backend, it returns the inputs, I didn't think about this solution